jx_sqlite/query_table.py

```diff
@@ -278,7 +278,7 @@ def query(self, query):
 
             output = Data(
                 meta={"format": "table"},
-                header=column_names,
+                header=[split_field(h)[0] if "\\" in h else h for h in column_names],
```

Comment:
be sure to pull upstream master and merge it to your local master.

I added `push_column_name` that may help with this `column_names` problem.  Well, by "help" I mean "provide a container for the data we will need to solve this problem.